boot/CMakeLists.txt

```diff
@@ -15,11 +15,12 @@ else()
     message(FATAL_ERROR "Unknown ARCH '" ${ARCH} "', cannot generate a valid UEFI boot filename.")
 endif()
 
-add_custom_target(efisys
```

Comment:
I suppose these modifications fix CMake dependencies?

Reply:
Not really. A custom target with a command is always considered out of date. Using the custom_command / custom_target couple makes the target up-to-date as long as its dependencies are up-to-date.

This avoids triggering mkisofs command each time you want to build the bootcd while it is already up-to-date.